index.d.ts

```diff
@@ -22,8 +22,7 @@ declare module '@primer/components' {
   }
 
   interface BorderProps
-    extends BaseProps,
-      StyledSystem.BordersProps,
+    extends StyledSystem.BordersProps,
       StyledSystem.BorderColorProps,
       StyledSystem.BoxShadowProps,
       StyledSystem.BorderRadiusProps {}
```

Comment:
`StyledSystem.BordersProps` includes `StyledSystem.BorderColorProps` and `StyledSystem.BorderRadiusProps`

```suggestion
    extends StyledSystem.BordersProps,
      StyledSystem.BoxShadowProps {}
```